test/lib/router.js

```diff
@@ -1421,6 +1421,26 @@ describe('Router', function () {
         done();
       });
     });
+
+    it('responds with 404 when has a trailing slash', function (done) {
```

Comment:
This test case asserts case-sensitivity, but the message here indicates that its testing a trailing slash. Could you correct either the message or the test case, whichever you intended?